Sources/MapboxMaps/Foundation/Camera/CameraAnimationDelegate.swift

```diff
@@ -6,14 +6,24 @@ public typealias AnimationCompletion = (UIViewAnimatingPosition) -> Void
 // MARK: CameraAnimatorDelegate Protocol
 internal protocol CameraAnimatorDelegate: class {
```

Comment:
let's move the `internal` designation onto each protocol member for consistency with what we'd decided as a team

Reply:
swiftlint complains because of the `explicit_top_level_acl` if we do this.

Reply:
Lets revisit this later when we figure out what to do with the lint rule.